### PR TITLE
[VIVO-1930] - Update default.runtime.properties and default.applicationSetup.n3

### PR DESCRIPTION
**Pull request by gneissone**
**[JIRA Issue](https://jira.lyrasis.org/browse/VIVO-1930)**: VIVO-1930
Companion pull request on the VIVO side is here: https://github.com/vivo-project/VIVO/pull/194

# What does this pull request do?
Improves example.runtime.properties so installation is easier for new users
Updates default database to TDB to match current best practices used for VIVO

# What's new?
1. Changes a couple default values in example.runtime.properties so first startup is smoother. 
&nbsp;&nbsp;&nbsp;&nbsp;- Change default Solr URL to match most common installation URL
&nbsp;&nbsp;&nbsp;&nbsp;- Remove email server defaults so VIVO doesn't throw warning on first startup.
&nbsp;&nbsp;&nbsp;&nbsp;~~- Change default root user email address to match VIVO's (and documentation).~~
2. Changes default database to TDB (from SDB) as we have done for VIVO

# How should this be tested?
On a new Vitro install, you should be able to ~~copy example.runtime.properties into a runtime.properties file and VIVO should~~ start up without errors or warnings (assuming you are using TDB and Solr exists at the typical location).

# Interested parties
@VIVO-project/vivo-committers

### CODE REVIEW

**Review by awoods — CHANGES_REQUESTED** [review]
Probably should update the text around "default":
https://github.com/vivo-project/Vitro/pull/190/files#diff-69f763958745df7a6600e5b4fe060cf7R85

See: https://github.com/vivo-project/VIVO/pull/193/files

**Comment by gneissone** [comment]
> Probably should update the text around "default":
> https://github.com/vivo-project/Vitro/pull/190/files#diff-69f763958745df7a6600e5b4fe060cf7R85
> 
> See: https://github.com/vivo-project/VIVO/pull/193/files

@awoods Thanks, updated the text.

**Comment by gneissone** [comment]
@awoods We're ready for re-review

File: home/src/main/resources/config/default.runtime.properties [hunk 1]
```diff
@@ -28,8 +28,8 @@ vitro.local.solr.url = http://localhost:8983/solr/vitrocore
 #     email.smtpHost = smtp.mydomain.edu
 #     email.replyTo = vitroAdmin@mydomain.edu
 #
-email.smtpHost =
-email.replyTo =
+# email.smtpHost =
```
Comment (awoods) [inline home/src/main/resources/config/default.runtime.properties:31]: We should align: https://github.com/vivo-project/VIVO/pull/222/commits/1ca40cd7b0703ac60fe938cf37a9b16c8501d2c7

Comment (gneissone) [reply home/src/main/resources/config/default.runtime.properties]: Getting closer to alignment... if you remove the ['Example' line](https://github.com/vivo-project/VIVO/pull/222/files#diff-863e26484eafb9690e1a9e118277d6e69045d655a07f0587b22ffacead7cbb2cR54) we will be there. I was following the format for the Solr variable below 🤷

**Review by awoods — APPROVED** [review]

File: home/src/main/resources/config/default.runtime.properties (hunk 1 above)
Comment (awoods) [reply home/src/main/resources/config/default.runtime.properties]: :link:

**Comment by gneissone** [comment]
Superseded by https://github.com/vivo-project/Vitro/pull/229

**Pull request closed without merging**